ctc_lib/logging/log_formatter.go

```diff
@@ -0,0 +1,107 @@
+/*
+Copyright 2018 Google Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package logging
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	nocolor = 0
+	red     = 31
+	green   = 32
+	yellow  = 33
+	blue    = 36
+	gray    = 37
+)
+
+// This Files Defines a LogFormatter With . This Format used is
+//<Level>: <Message>
+
+type CTCLogFormatter struct {
+	EnableColors bool
+}
+
+func NewCTCLogFormatter(enableColors bool) *CTCLogFormatter {
+	return &CTCLogFormatter{
+		EnableColors: enableColors,
+	}
+}
+
+func (f *CTCLogFormatter) Format(entry *log.Entry) ([]byte, error) {
```

Comment:
Is this defined by a logrus interface? If so, could you leave a comment pointing to that type.

Reply:
Done.